imports/ui/components/news/news_list.js

```diff
@@ -42,3 +42,4 @@ Template.newsList.events({
     }, 1000);
   }
 });
+*/
```

Comment:
In general, we should delete blocks that have been commented out unless there is a reason to keep them (ie. we are going to bring them back, etc.), in which case we should include a comment about why they've been kept.